run/services/slack/surfaces/modals/scalingo-apps/application-creation.js

```diff
@@ -0,0 +1,44 @@
+const { Modal, Blocks, Elements, Bits } = require('slack-block-builder');
+const regions = [
+  { id: 'osc-secnum-fr1', name: 'production' },
+  { id: 'osc-fr1', name: 'recette' },
+];
```

Comment:
**nitpick**
Extraire dans `config.js`
``suggestion
const regions = [
  { id: 'osc-secnum-fr1', name: 'securized' },
  { id: 'osc-fr1', name: 'not-securized' },
];
```

Reply:
Encore mieux, on peut récupérer la liste des régions via le client scalingo
https://github.com/Scalingo/scalingo.js/blob/16e4fbb6daa90aac951fc730e8525d199a42195f/src/Regions/index.ts
Cependant, Il faudrait mettre à jour le client node donc peut être à faire dans une prochaine PR

Reply:
J'ai cependant remplacer les noms par ça : 
"name" : "osc-fr1" > "Paris - Outscale",
"name" : "osc-secnum-fr1"  >  "Paris - SecNumCloud - Outscale",